elk/training/ccs.py

```diff
@@ -85,6 +90,9 @@ def reset_parameters(self):
             for layer in self.probe:
                 if isinstance(layer, nn.Linear):
                     layer.reset_parameters()
+        elif self.init == "zero":
```

Comment:
Just curious, why did you add this?

Reply:
I noticed that you tend to get non-trivial probes with high accuracy even without any confidence loss, and I wanted to test the hypothesis that it's due to an initialization issue where it's hard for the optimizer to get to the trivial solution of outputting 0.5 all the time. Turns out to be true. See messages in the ELK channel a couple days ago